src/idioms/mem-replace.md

```diff
@@ -92,8 +91,7 @@ borrow checker. The compiler may fail to optimize away the double store,
 resulting in reduced performance as opposed to what you'd do in unsafe
 languages.
 
-Furthermore, the type you are taking needs to implement the [`Default`
-trait](./default.md). However, if the type you're working with doesn't
+Furthermore, the type you are taking needs to implement the [`Default` trait](./default.md). However, if the type you're working with doesn't
```

Comment:
this line is longer than 80 characters. Why isn't dprint complaining?
Is the ci check working?